docs/modules/ROOT/partials/getting-started/con-registry-serdes-strategy.adoc

```diff
@@ -48,31 +48,48 @@ However, you can use alternative conventions for the mapping by using a strategy
 
 
 [discrete]
-== Global ID strategy
-
-The global ID strategy locates and identifies the specific version of the schema registered under the artifact ID provided by the artifact ID strategy. Every version of every artifact has a single globally unique identifier that can be used to retrieve the content of that artifact. This global ID is included in every Kafka message so that a deserializer can properly fetch the schema from {registry}.  
+== DefaultSchemaResolver interface
 
-The global ID strategy can look up an existing artifact version, or it can register one if not found, depending on which strategy is used. You can also provide your own strategy by creating a
-custom Java class that implements `io.apicurio.registry.utils.serde.strategy.GlobalIdStrategy`.
+The default schema resolver locates and identifies the specific version of the schema registered under the artifact reference provided by the artifact resolver strategy. Every version of every artifact has a single globally unique identifier that can be used to retrieve the content of that artifact. This global ID is included in every Kafka message so that a deserializer can properly fetch the schema from Apicurio Registry.
+ 
+The default schema resolver can look up an existing artifact version, or it can register one if not found, depending on which strategy is used. You can also provide your own strategy by creating a custom Java class that implements `io.apicurio.registry.serde.SchemaResolver`. However, it is recommended to use the `DefaultSchemaResolver` and specify configuration properties instead.
 
 [discrete]
-[id='service-registry-concepts-globalid-{context}']
+[id='configuring-globalid-strategy-{context}']
 == Strategies to return a global ID
 
-{registry} provides the following strategies to return a global ID based on an implementation of `GlobalIdStrategy`:
+When using the `DefaultSchemaResolver`, you can configure the following application properties for the global ID strategy: 
```

Comment:
I would change "for the global ID strategy" with something like "to adjust the schema resolver behavior to your needs" .